market_maker_keeper/reloadable_config.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, filename: str):
         assert(isinstance(filename, str))
 
         self.filename = filename
+        self._checksum = None
```

Comment:
Perhaps this should be named to indicate it's the checksum only for token configuration.

Reply:
That makes sense, I will update the name.